wandb/interface/interface.py

```diff
@@ -515,10 +515,7 @@ def _communicate_run(self, run, timeout=None):
         req = self._make_record(run=run)
         resp = self._communicate(req, timeout=timeout)
         if resp is None:
-            # TODO: friendlier error message here
-            raise wandb.Error(
-                "Couldn't communicate with backend after %s seconds" % timeout
-            )
+            return
```

Comment:
This is handled by callers of communicate_run()

Reply:
Only nit would be to add a comment here saying where this is handled.

Reply:
added